concent_api/verifier/tasks.py

```diff
@@ -286,6 +290,10 @@ def blender_verification_order(
             generate_verifier_storage_file_path(result_package_path)
         )
 
+        # OpenCV is imported here because it requires OpenCV package installed in the system to work,
+        # which is not installed on Concent instances without verifier.
+        cv2 = import_cv2()
```

Comment:
why not just `import cv2` here (local import)?

Reply:
ok, I know why - for patching the tests

Reply:
@kbeker: If @dybi has to ask - it's a good sign that you should add a comment with an explanation here.

You do have a comment but it's actually @rwrzesien's comment moved from the top of the file without any change. It should be expanded.

Reply:
:+1: done